README.md

```diff
@@ -23,36 +23,120 @@ The program is currently in production, getting live feedback.
 ## Contributing to Ready Responder
 We have a Slack channel at [readyresponder.slack.com](https://readyresponder.slack.com) to give help if you need it.
 
-### Getting Started
+### Getting Started - Dependencies
 
-This is a Rails project that is configured to run on Ruby 2, and on a Postgres database.  So, the things you'll need to install before running ReadyResponder locally are Ruby, the `bundler` gem, and Postgres version 9.
+This is a Rails project that is configured to run on Ruby 2, and on a Postgres
+database. So, the things you'll need to install before running ReadyResponder locally are:
 
-1. Ruby: There's a detailed list of options for installing Ruby on the [official Ruby website](https://www.ruby-lang.org/en/documentation/installation/).
-  * [RVM](http://rvm.io/), [rbenv](https://github.com/rbenv/rbenv#readme), and [chruby](https://github.com/postmodern/chruby#readme) are common ruby installation managers for Macs & Linux.
-  * The exact version of Ruby that ReadyResponder is using is specified in the [.ruby-version](.ruby-version) file.
-2. Bundler: `gem install bundler`
-3. Postgres
-  * If you have `homebrew` on a Mac, you can run `brew install postgres`.
-  * Alternatively, [Postgres.app](http://postgresapp.com) is an easy way to get started with PostgreSQL on the Mac. [PostgresApp 9.4.8](https://github.com/PostgresApp/PostgresApp/releases/tag/9.4.8) has been tested (when configuring, add `host: localhost` to `config/database.yml`).
+* Ruby
+* the `bundler` gem
+* Postgres version 9, the database used to store the application data
+* SQLite, a dependency of the mailcatcher gem
+* ImageMagick, a dependency of the rmagick gem, used to process images
+
+For **ruby**, you can find a detailed list of options on the [official Ruby website](
+https://www.ruby-lang.org/en/documentation/installation/). The most common
+applications used to manage your ruby version are:
+* [RVM](https://rvm.io)
+* [rbenv](https://github.com/rbenv/rbenv#readme), with the
+  [ruby-build](https://github.com/rbenv/ruby-build#readme) plugin
+* [chruby](https://github.com/postmodern/chruby#readme)
+
+After setting up ruby on your system, install the `bundler` gem with `gem
```

Comment:
I wonder if this line regarding bundler is out of place. Chris H's instructions needed bundler to happen after RMagick, but this would put it before.

Reply:
In here we're only installing the bundler gem. ImageMagick must be available before running `bundle install`, which is run later on during the project setup.